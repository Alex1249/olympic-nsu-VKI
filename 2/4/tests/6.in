27
0 0
1 0
1 1
2 0
2 1
2 2
3 0
3 1
3 2
3 3
999 0
999 999
999 499
999 500
1000 0
1000 100
1000 200
1000 300
1000 400
1000 500
1000 600
1000 700
1000 800
1000 900
1000 1000
1000 1
1000 999
