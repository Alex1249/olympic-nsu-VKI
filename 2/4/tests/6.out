1
1
1
1
2
1
1
3
3
1
1
1
1.35144120472718e+299
1.35144120472718e+299
1
6.38505119263052e+139
6.61715556065931e+215
5.42825004640614e+263
4.96527238625423e+290
2.70288240945437e+299
4.96527238625423e+290
5.42825004640614e+263
6.61715556065931e+215
6.38505119263052e+139
1
1000
1000
